### ISSUE TITLE
Paper interpretation

### ISSUE THREAD

**Issue by Xiaozhi-sudo** [dvtailor/meta-l2d#3]
Dear author, I am a little confused about some of the content in the paper.In the third section of the third chapter, why can't multi-expert L2D be applied to L2D-Pop? Isn't it simply a matter of modifying the output layer and the loss function of the model? I don't quite understand this, and I would appreciate your explanation.

**Comment by dvtailor** [comment]
Hi, thanks for your interest in our paper. The main reason is that multi-expert L2D cannot cope with unseen experts at test-time. As you correctly remark in multi-expert L2D, the output space is augmented with the number of available experts at train-time but it is assumed the same experts are available at test-time. However if one can come up with a way to match (possibly novel) test-time experts to those observed during training then it is not unreasonable to expect that a modified form of multi-expert L2D could be applied in the population setting.
